src/finch/tensor.py

```diff
@@ -16,17 +15,36 @@ def __str__(self):
 
 
 class Tensor(_Display):
-    def __init__(self, lvl=None, arr=None, jl_data=None):
+    """
+    A wrapper class for Finch.Tensor and Finch.SwizzleArray
+    """
+    ROW_MAJOR = "C"
+    COLUMN_MAJOR = "F"
+
+    def __init__(self, lvl=None, arr=None, jl_data=None, order=None):
         if arr is not None and not isinstance(arr, np.ndarray):
-            raise ValueError("For now only numpy input allowed")
+            raise ValueError("For now only numpy input allowed.")
+        if order not in (None, self.ROW_MAJOR, self.COLUMN_MAJOR):
+            raise ValueError(f"order must be a 'C' or 'F', but is: {type(order)}.")
```

Comment:
Let's support arbitrary orders too as an array of dims

Reply:
If we want to support arbitrary orders, we can by manually calling `.transpose(axis=...)` on the constructed object; does it make sense to put that into the constructor itself?

Reply:
FYI I just pushed changes that support arbitrary orders. I will explain that in a separate comment.

Reply:
Right now any order that is a permutation can be passed (and also string values "F" and "C"). `order` remembers that permutation and once we want to move back to numpy with `todense()` it's just a `.transpose(self.get_inv_order())` to move back to numpy order.